testing/testing.mk

```diff
@@ -154,7 +154,7 @@ $(eval $(call mldb_unit_test,MLDB-723-jseval-exceptions.js))
 $(eval $(call mldb_unit_test,MLDB-761-sub-queries.py))
 $(eval $(call mldb_unit_test,MLDB-565-classifier-details.js))
 $(eval $(call mldb_unit_test,MLDB-1597-regression.py))
-$(eval $(call mldb_unit_test,MLDB-749-csv-dataset.js))
+$(eval $(call mldb_unit_test,MLDB-749-csv-dataset.js,,manual)) #reading from old user
```

Comment:
Looks like this could be a misleading error message while I was testing that everything runs with the Structured dataset. I'll investigate.